doc/developer/testdrive.md

```diff
@@ -407,6 +407,22 @@ $ set schema={
     }
 ```
 
+#### `$ set-arg-default name=value`
+
+Sets the default value for an argument variable passed on the command line. If
+`--var=name=cli-value` is specified on the command line, it will take precedence
+over the argument default.
+
+Note that the variable name should not include the `arg.` prefix. Testdrive will
```

Comment:
Good to mention it here (but it would not work with `arg.` because testdrive will be confused by the dot; that's why I did not add a check in the code 😀).

Reply:
Yes, makes sense!